pkg/geoipupdate/database/http_reader.go

```diff
@@ -73,6 +73,9 @@ func (r *HTTPReader) Read(ctx context.Context, editionID, hash string) (*ReadRes
 	err = internal.RetryWithBackoff(
 		func() error {
 			result, err = r.get(ctx, editionID, hash)
+			if r.verbose && err != nil {
+				log.Printf("Couldn't download %s, retrying: %v", editionID, err)
+			}
```

Comment:
Hmm, this will say we are retrying even if we aren't retrying. This might be particularly confusing on 4xx errors where we don't retry at all. Two possible solutions for this:

1. Switch to `github.com/cenkalti/backoff/v4` and get rid of our internal `RetryWithBackoff`. It has a `RetryNotify` function that would work here. I am not sure why we wrote our own, maybe just to avoid the extra dependency. @horgh might recall the history with that.
2. Modify `internal.RetryWithBackoff` to take a notify function that is called with the error right before the `time.Sleep`.

Reply:
Good point. I've replaced the internal implementation with `cenkalti/backoff`.